lib/StaticNarrative/exporter/exporter.py

```diff
@@ -102,9 +104,12 @@ def _build_exporter(self, exported_data: dict, ws_id: int) -> HTMLExporter:
             netloc = "narrative." + netloc
         host = (endpt_parsed.scheme or "https") + "://" + netloc
 
-        c.TemplateExporter.template_path = [
-            ".",
-            os.path.join(base_path, "static", "templates"),
+        tpl_base_dir = os.path.join(STATIC_NARRATIVE_BASE_DIR, "lib", "StaticNarrative", "exporter", "static", "templates")
+        print("template base dir: " + tpl_base_dir)
+        c.TemplateExporter.template_paths = [
+            tpl_base_dir,
+            os.path.join(tpl_base_dir, "html"),
+            os.path.join(tpl_base_dir, "skeleton"),
```

Comment:
The updated version of nbformat requires the full template paths to be specified, and it also gets rid of the templates that came with nbformat v5. For now, I've just copied in the templates from v5, but when I start editing the templates themselves, I will see if we can do away with them and/or use replacements from the v7 nbformat.

Reply:
(removed the print statement, which has messed up the comments)